### PR TITLE
Update HV constructor

### PR DESCRIPTION
**Pull request by darrylmasson**
Support for https://github.com/XENONnT/daqnt/pull/33.

### CODE REVIEW

File: dispatcher/config.ini [hunk 1]
```diff
@@ -72,6 +72,14 @@ MasterDAQConfig = {
...
     }
 
+VMEConfig = {
+  "0": "192.168.131.60",
+  "1": "192.168.131.61",
+  "2": "192.168.131.62",
+  "3": "192.168.131.63",
+  "4": "192.168.131.64"
+}
+
```
Comment (jorana) [inline dispatcher/config.ini:82]: Why are the IPs different from https://github.com/XENONnT/daqnt/pull/33/files#diff-2dac8abc3e3c1e09e1e08cfc16f6954cc245484ea396708142dc1f29d6be07f5R31-R34

File: dispatcher/dispatcher.py [hunk 2]
```diff
@@ -36,15 +27,17 @@ def main():
...
     config = config['DEFAULT' if not args.test else "TESTING"]
     config['MasterDAQConfig'] = json.loads(config['MasterDAQConfig'])
-    control_uri = config['ControlDatabaseURI']%os.environ['MONGO_PASSWORD']
+    control_uri = config['ControlDatabaseURI']%quote_plus(os.environ['MONGO_PASSWORD_DAQ'])
     control_mc = pymongo.MongoClient(control_uri)
-    runs_uri = config['RunsDatabaseURI']%os.environ['RUNS_MONGO_PASSWORD']
+    runs_uri = config['RunsDatabaseURI']%quote_plus(os.environ['MONGO_PASSWORD_RUNS'])
     runs_mc = pymongo.MongoClient(runs_uri)
     logger = daqnt.get_daq_logger(config['LogName'], level=args.log, mc=control_mc)
+    vme_config = json.loads(config['VMEConfig'])
```
Comment (jorana) [inline dispatcher/dispatcher.py:35]: can we use https://github.com/XENONnT/daqnt/pull/36/files

File: dispatcher/config.ini (hunk 1 above)
Comment (darrylmasson) [reply dispatcher/config.ini:82]: These are the addresses of the 5 VME crates. I'm not sure why you're comparing the production addresses to testing addresses.

File: dispatcher/dispatcher.py (hunk 2 above)
Comment (darrylmasson) [reply dispatcher/dispatcher.py:35]: Of course

File: dispatcher/config.ini (hunk 1 above)
Comment (jorana) [reply dispatcher/config.ini:82]: oops 🤦

**Review by jorana — APPROVED** [review]
nice 🎉

**Pull request merged**